Package.swift

```diff
@@ -21,7 +21,7 @@ let swiftSystem: PackageDescription.Target.Dependency = .product(name: "SystemPa
 
 // These platforms require a dependency on `NIOPosix` from `NIOHTTP1` to maintain backward
 // compatibility with previous NIO versions.
-let historicalNIOPosixDependencyRequired: [Platform] = [.macOS, .iOS, .tvOS, .watchOS, .linux, .android]
+let historicalNIOPosixDependencyRequired: [Platform] = [.macOS, .iOS, .tvOS, .watchOS, .linux, .android, .custom("freebsd"), .custom("FreeBSD")]
```

Comment:
These platforms do not require a historical NIOPosix dependency, as they did not compile before.